src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -85,6 +85,8 @@ internal bool IsReplaying
 
         internal bool IsCompleted { get; set; }
 
+        internal bool IsLongRunningTimer { get; set; }
```

Comment:
Can we make this `private set` instead of `set`? We should always try to minimize the visibility of fields where reasonably possible.